intermediate-python-debugging/vs-code-debugging.md

```diff
@@ -0,0 +1,190 @@
+# Debugging With VS Code
```

Comment:
My ideal is either:

"Debugging with VS Code" and file name `debugging-with-vs-code`
or "VS Code Debugging" and file name `vs-code-debugging`

Reply:
Done